presto-hive/src/test/java/com/facebook/presto/hive/metastore/glue/TestHiveClientGlueMetastore.java

```diff
@@ -174,6 +178,16 @@ protected ExtendedHiveMetastore createMetastore(File tempDir)
         return new GlueHiveMetastore(hdfsEnvironment, glueConfig, executor);
     }
 
+    @Override
+    protected Set<HiveStorageFormat> getSupportedCreateTableHiveStorageFormats()
+    {
+        return difference(
+                ImmutableSet.copyOf(super.getSupportedCreateTableHiveStorageFormats()),
+                // Exclude PAGEFILE because Glue catalog requires a non-empty SerDe and PageFile has it set to an
+                // empty string.
+                ImmutableSet.of(PAGEFILE));
```

Comment:
Just a minor comment, we should also exclude AVRO and CSV here as they have been excluded in the parent class `AbstractTestHiveClient`

Reply:
I am calling `super.getSupportedCreateTableHiveStorageFormats` from the parent class so those file formats will already be excluded.

Reply:
Oh my bad, missed that.

Reply:
It is all good, no worries. Let me know if there is anything else I should address in this PR, thanks!

Reply:
LGTM 👍🏼